tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py

```diff
@@ -248,8 +248,9 @@ def index_map_nchw32c_nchw8h8w32c(n_batch, channel, height, width, channel_32):
 
         # Add cache for input and output activation layout transform,
         # note that weight is already in correct layout
-        input_cache = sch.cache_read(conv2d_block, 0, "global")  # pylint: disable=unused-variable
-        output_cache = sch.cache_write(outer_block, 0, "global")  # pylint: disable=unused-variable
+        # pylint: disable=unused-variable
```

Comment:
Put this at the top of the file (see other files disabling pylint) if you need it.

Reply:
This was needed only for `input_cache` and `output_cache` variables. I will move it to the top in next PR.